### ISSUE TITLE
Namespace not specified

### ISSUE THREAD

**Issue by erperejildo** [pikaju/flutter-maps-launcher#44]
```
A problem occurred configuring project ':maps_launcher'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file
```

**Comment by sarahk** [comment]
Is there an ETA for pushing this fix?

**Comment by pauljonescodes** [comment]
This is also preventing me from using this plugin :-(

**Comment by maquejp** [comment]
Same here ... I am still using Koala patch 1 version of Android Studio


Execution failed for task ':maps_launcher:packageDebugResources'.
> Error while evaluating property 'namespace' of task ':maps_launcher:packageDebugResources'.
   > Failed to calculate the value of task ':maps_launcher:packageDebugResources' property 'namespace'.
      > Failed to calculate the value of property 'namespace'.
         > Package Name not found in /home/devlin/.pub-cache/hosted/pub.dev/maps_launcher-2.2.1/android/src/main/AndroidManifest.xml, and namespace not specified. Please specify a namespace for the generated R and BuildConfig classes via android.namespace in the module's build.gradle file like so:

           android {
               namespace 'com.example.namespace'
           }

**Comment by paul-charlton** [comment]
Is there an updated on releasing this?  It looks like there's a PR with the required changes.

Currently going to look at alternatvies...

**Comment by igoriuz** [comment]
As a hotfix, you could add the following lines to `android/build.gradle` within `allProjects`

```
allprojects {
    repositories {
        google()
        mavenCentral()
    }
    // This code is where all the magic happens and fixes the error.
    subprojects {
        afterEvaluate { project ->
            if (project.hasProperty('android')) {
                project.android {
                    if (namespace == null) {
                        namespace project.group
                    }
                }
            }
        }
    }
    // This code is where all the magic happens and fixes the error.
}
```

Source: https://stackoverflow.com/a/77738936

**Comment by pikaju** [comment]
Hey, sorry for the wait. This problem should be fixed in `3.0.0`, could you check if it works? If not, please open another issue.
